test-case/check-playback.sh

```diff
@@ -94,6 +104,17 @@ do
                     dmesg > $LOG_ROOT/aplay_error_${dev}_$i.txt
                     dloge "aplay on PCM $dev failed at $i/$loop_cnt."
                     exit 1
+                else
+                    #sleep random_min ~ random_max seconds before running aplay, then aplay can be run under dsp active or suspended status randomly
+                    if [[ $(sof-dump-status.py --dsp_status 0) != "unsupported" ]] && [ ${OPT_VALUE_lst['R']} -eq 1 ]; then
+                        time=$(func_lib_get_random $random_min $random_max)
```

Comment:
can you rebase your code: https://github.com/thesofproject/sof-test/blob/master/case-lib/lib.sh#L121

```suggestion
                        time=$(func_lib_get_random $random_max $random_min)
```